### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Bugses**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,2 @@
-# GameFall2022
\ No newline at end of file
+# GameFall2022
+nasD IOPnaipsod npisd aopniSA DOnpioASD Fnpisa d
```
Comment (brendanehrman1) [inline README.md:2]: this doesn't make sense

**Comment by Bugses** [comment]
no it does

**Comment by Bugses** [comment]
asd asd asd

**Review by brendanehrman1 — CHANGES_REQUESTED** [review]
interesting changes lol

**Pull request closed without merging**